src/interfaces/Alert.ts

```diff
@@ -1,41 +1,12 @@
 export default interface Alert {
-    _id: string;
-    alertId: string;
     message: string;
-    details: {
-        Name: string;
-        TaskName: string;
-        Group: string;
-        Tags: {
-            [key: string]: string;
-        };
-        ServerInfo: {
-            Hostname: string;
-            ClusterID: string;
-            ServerID: string;
-        };
-        ID: string;
-        Fields: {
-            [key: string]: number;
-        };
-        Level: string;
-        Time: string;
-        Duration: number
-        Message: string;
-    };
     time: string;
-    duration: number;
     level: string;
-    data: {
-        series: {
-            name: string;
-            tags: {
-                [key: string]: string;
-            };
-            columns: string[];
-            values: string[];
-        };
+    backgroundColor: string;
+    fqdn: string;
+    serverInfo: {
```

Comment:
I'm not sure, what about the `device` information we have for the `disk` measurement? Is it enough to have it in the alert message and not (optional) in the server info?

Reply:
We should discuss that IRL.